docs/docs/controls/c-0221.md

```
@@ -3,7 +3,7 @@
 [← Back to all controls](index.md)
 
 
-_Integrate with cloud provider (see [here](../../integrations/kubescape-integration-with-cloud-providers))_
+_Integrate with cloud provider (see [here](../integrations/kubescape-integration-with-cloud-providers))_
```

markdownlint-cli2 (0.23.2)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)